src/gui/transferlistwidget.cpp

```diff
@@ -807,6 +812,38 @@ void TransferListWidget::askAddTagsForSelection()
         addSelectionTag(tag);
 }
 
+void TransferListWidget::editTorrentTrackers()
+{
+    const QVector<BitTorrent::TorrentHandle *> torrents = getSelectedTorrents();
+    QVector<BitTorrent::TrackerEntry> commonTrackers;
+
+    if (!torrents.empty()) {
+        commonTrackers = torrents[0]->trackers();
+
+        for (const BitTorrent::TorrentHandle *torrent : torrents) {
+            QSet<BitTorrent::TrackerEntry> trackerSet;
+
+            for (const BitTorrent::TrackerEntry &entry : asConst(torrent->trackers()))
+                trackerSet.insert(entry);
+
+            commonTrackers.erase(std::remove_if(commonTrackers.begin(), commonTrackers.end()
+                , [&trackerSet](const BitTorrent::TrackerEntry &entry) { return !trackerSet.contains(entry); })
+                , commonTrackers.end());
+        }
+    }
+
+    auto trackerDialog = new TrackerEntriesDialog(this);
+    trackerDialog->setAttribute(Qt::WA_DeleteOnClose);
+    trackerDialog->setTrackers(commonTrackers);
+
+    connect(trackerDialog, &QDialog::accepted, this, [torrents, trackerDialog]() {
```

Comment:
Coding Style issue. Move brace to the next line.

Reply:
fixed.